alma/lib/Model/PaymentData.php

```diff
@@ -220,4 +240,13 @@ public static function dataFromCart($cart, $context, $feePlans, $forPayment = fa
 
         return $dataPayment;
     }
+
+    private function getNewCustomer($idCustomer)
```

Comment:
A method that is starts with `get` (a getter) should return something.
A method that is return a `bool` should starts with `is` or `has`.

`getNewCustomer($id)` should retrieve a new customer by the given `$id`

Maybe the method should be called `isNewCustomer` ?